editor/src/java/com/defold/editor/providers/JsonProviders.java

```diff
@@ -127,15 +127,15 @@ private static Object JsonToObject(JsonNode node, Message.Builder builder, Field
 
             case VALUE_NUMBER_FLOAT:
                 if (type == FieldDescriptor.JavaType.FLOAT)
-                    return (float) node.getValueAsDouble();
+                    return (float) node.asDouble();
                 else if (type == FieldDescriptor.JavaType.DOUBLE)
-                    return node.getValueAsDouble();
+                    return node.asDouble();
 
             case VALUE_NUMBER_INT:
                 if (type == FieldDescriptor.JavaType.INT)
-                    return node.getValueAsInt();
+                    return node.asInt();
                 else if (type == FieldDescriptor.JavaType.LONG)
-                    return node.getValueAsLong();
+                    return node.asLong();
```

Comment:
Fixed deprecation warnings after adding `:javac-options ["-Xlint:unchecked" "-Xlint:deprecation"]` to `project.clj`